include/linker_set.h

```diff
@@ -46,6 +46,9 @@
 #define SET_FOREACH(pvar, set)                                                 \
   for (pvar = SET_BEGIN(set); pvar < SET_LIMIT(set); pvar++)
 
+#define SET_FOREACH_PTR(pvar, set_b, set_e)                                    \
```

Comment:
`set_b`, `set_e` - set begin and set end ? Please make the naming more verbose so that we don't have to guess, preferably just `set_begin`, `set_end`. Also a short docstring about when I would want to use this one over `SET_FOREACH` could be helpful

Reply:
done